### PR TITLE
Add ci job for nutanix disconnected install on 4.12

### PR DESCRIPTION
**Pull request by sgaoshang**

### CODE REVIEW

**Comment by sgaoshang** [comment]
periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-nutanix-ipi-proxy-fips-p2-f28

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-nutanix-ipi-proxy-fips-p2-f28

**Comment by jianlinliu** [comment]
cc @liangxia to review.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 1]
```diff
@@ -2628,6 +2628,57 @@ tests:
...
+    cluster_profile: nutanix-qe-dis
+    env:
+      E2E_RUN_TAGS: '@amd64 and @nutanix-ipi and @disconnected and @network-ovnkubernetes
+        and not @fips and @destructive'
+      TAG_VERSION: '@4.12'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
+    test:
+    - chain: openshift-e2e-test-qe-destructive
+    workflow: cucushift-installer-rehearse-nutanix-ipi-disconnected
+- as: nutanix-ipi-proxy-fips-p2-f28
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:2657]: I think this one can cover `nutanix-ipi-fips-p2-f7`, perhaps merge them ?

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:2657]: Good point.

So my proposal is :
1. merging them
2. remove `fips` from `nutanix-ipi-fips-p2-f7` to `nutanix-ipi-p2-f28`

**Pull request closed without merging**